docs/v0.9/zh/getting-started/installation/greptimedb-cluster.md

```diff
@@ -2,68 +2,153 @@
+### 前置条件
+
+使用 Docker Compose 是运行 GreptimeDB 集群的最简单方法。开始之前，请确保已经安装了 Docker。
+
+### 步骤 1: 下载 Docker Compose 的 YAML 文件
 
 ```
-curl -fsSL https://downloads.greptime.cn/releases/scripts/gtctl/install.sh | sh -s -- -s aws
+wget https://raw.githubusercontent.com/GreptimeTeam/greptimedb/<%greptimedb-version%>/docker/docker-compose/cluster-with-etcd.yaml
 ```
 
-:::tip Note
+### 步骤 2: 启动集群
 
-若您使用 Windows 操作系统，考虑到各组件运行的复杂性和兼容性，我们强烈建议您开启 WSL([Windows Subsystem for Linux](https://learn.microsoft.com/en-us/windows/wsl/about))，启动一个最新的 Ubuntu 来继续执行 GreptimeDB 集群版的安装。
+```
+GREPTIMEDB_VERSION=<%greptimedb-version%> \
+GREPTIMEDB_REGISTRY=greptime-registry.cn-hangzhou.cr.aliyuncs.com \
+ETCD_REGISTRY=greptime-registry.cn-hangzhou.cr.aliyuncs.com \
+  docker compose -f ./cluster-with-etcd.yaml up 
+```
 
-:::
+如果集群成功启动，它将监听 4000-4003 端口。你可以通过参考 [快速开始](../quick-start.md#连接到-greptimedb) 访问集群。
 
-一旦下载已经完成，`gtctl` 的二进制文件将保存在你的当前目录下。
+### 清理
 
-最快体验 GreptimeDB 分布式集群的方式便是运行 `playground` 命令：
+你可以使用以下命令停止集群：
 
 ```
-./gtctl playground
+docker compose -f ./cluster-with-etcd.yaml down
 ```
 
-当命令执行完毕之后，playground 将会在前台启动。你可以使用 `Ctrl+C` 来停止 playground。playground 将会在你的主机上以 bare-metal 模式部署最小的 GreptimeDB 集群。
-
-如果一切正常，你可以使用浏览器访问内置的 Dashboard `http://localhost:4000/dashboard/`。
+默认情况下，数据将被存储在 `/tmp/greptimedb-cluster-docker-compose`。如果你想清理数据，也可以删除数据目录：
 
-更多的细节，请参考 [gtctl operations](/reference/gtctl.md)。
+```
+rm -rf /tmp/greptimedb-cluster-docker-compose
+```
 
 ## 在 Kubernetes 中部署 GreptimeDB 集群
 
-如果您的 Kubernetes 集群已经准备就绪（需要 1.18 或更高版本），您可以使用以下命令部署 GreptimeDB 集群：
+我们将使用 [GreptimeTeam/helm-charts](https://github.com/GreptimeTeam/helm-charts) 在 Kubernetes 中部署 GreptimeDB 集群。我们建议在 Kubernetes 中部署 GreptimeDB 以供生产环境使用。
+
+### 前置条件
+
+- Kubernetes >= 1.18
+
+  :::tip 注意
+
+  你可以使用 [kind](https://kind.sigs.k8s.io/docs/user/quick-start/) 来创建一个本地 Kubernetes 集群进行测试：
+
+  ```
+  kind create cluster
+  ```
+
+  :::
+
+- 安装 [`helm`](https://helm.sh/docs/intro/install/)
+
+- 安装 [`kubectl`](https://kubernetes.io/docs/tasks/tools/)
+
+### 步骤 1: 添加 Chart 仓库
+
+你可以使用以下命令添加 Chart 仓库：
 
 ```
-./gtctl cluster create mycluster --use-greptime-cn-artifacts
+helm repo add greptime https://greptimeteam.github.io/helm-charts/
+helm repo update
 ```
 
-创建完成后，你可以使用以下命令连接集群：
+你可以运行以下命令查看 Charts：
 
 ```
-./gtctl cluster connect mycluster
+helm search repo greptime
 ```
 
-您还可以使用 [Helm Charts](/user-guide/operations/deploy-on-kubernetes/overview.md) 来部署集群。
-
-:::tip 提示
+### 步骤 2: 安装 etcd 集群
 
-您可以使用 [kind](https://kind.sigs.k8s.io/docs/user/quick-start/) 来创建 Kubernetes：
+我们推荐使用 Bitnami etcd chart 来部署 etcd 集群：
 
 ```
-kind create cluster
+helm upgrade \
+  --install etcd oci://registry-1.docker.io/bitnamicharts/etcd \
+  --set image.registry=greptime-registry.cn-hangzhou.cr.aliyuncs.com \
+  --set image.tag=3.5.12 \
+  --set replicaCount=3 \
+  --set auth.rbac.create=false \
+  --set auth.rbac.token.enabled=false \
+  --create-namespace \
+  -n etcd-cluster
 ```
```

Comment:
We should use CN region charts avoid network fail, same as: https://docs.greptime.cn/user-guide/operations/deploy-on-kubernetes/deploy-greptimedb-cluster